metadata_proxy.go

```diff
@@ -43,17 +44,20 @@ var (
 )
 
 var (
+	addr                = flag.String("addr", "127.0.0.1:988", "address at which to listen and proxy")
+	metricsAddr         = flag.String("metrics_addr", "127.0.0.1:989", "address at which to publish metrics")
```

Comment:
Nit: I had the impression that in Kubernetes we use dashes in flags and capitalize help strings, see e.g. https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/options/options.go

Reply:
Done.